cdap-ui/app/cdap/components/AdminDetailPanel/index.js

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright © 2016 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import MetadataPaneApplication from '../MetadataPaneApplication/index.js';
+import MetadataPaneMemory from '../MetadataPaneMemory/index.js';
+import MetadataPaneNode from '../MetadataPaneNode/index.js';
+import MetadataPaneVirtualCores from '../MetadataPaneVirtualCores/index.js';
+
+import React, {PropTypes} from 'react';
+require('./AdminDetailPanel.less');
+
+const propTypes = {
+  applicationName: PropTypes.string,
+  timeFromUpdate: PropTypes.number,
+  isLoading: PropTypes.bool
+};
+
+function AdminDetailPanel({applicationName, timeFromUpdate, isLoading}){
+  return (
+    <div className="admin-detail-panel">
+      <div className="admin-detail-panel-button-left">
+        <i className="fa fa-chevron-left" aria-hidden="true" />
+      </div>
+      <div className="admin-detail-panel-button-right">
+        <i className="fa fa-chevron-right" aria-hidden="true" />
+      </div>
+      <div className="admin-detail-panel-header">
+        <div className="admin-detail-panel-header-name">
+          {applicationName}
+        </div>
+        <div className="admin-detail-panel-header-status">
+          Last updated {timeFromUpdate} seconds ago
+        </div>
+      </div>
+      <div className="admin-detail-panel-body">
+        <div className="end-line" />
+        <MetadataPaneNode isLoading={isLoading} label="Nodes" numStats={6} />
```

Comment:
We talked about `numStats` right? We don't need numStats for MetadataPaneNode I believe. When we talked about it we said `<AdminNodesMetadata />` will be a smart component and knows what stats to display. So just dropping in the component should be add things.